boards/arm/stm32h735g_disco/Kconfig.defconfig

```diff
@@ -12,5 +12,7 @@ config SPI_STM32_INTERRUPT
 	default y
 	depends on SPI
 
+config STM32_LPTIM_TIMER
+	default y if PM
```

Comment:
Move to soc/arm/st_stm32/stm32h7/Kconfig.defconfig.series

Reply:
done